update to parts list

thewaluigi opened the pull request.

thewaluigi committed:
Update PartsList.md

thewaluigi committed:
Merge pull request #1 from thewaluigi/pneumatic-update--patch-1

Update PartsList.md

RandalMcAllister reviewed (changes requested):
Correct spelling errors

thewaluigi committed:
parts list 

this is for eletrical and numatic parts for  den parts
https://docs.google.com/spreadsheets/d/1vCiIbkgaA1AcBXfUxXR7KG62Gx-o_6eUTM6lcIw3-O8/edit#gid=0

thewaluigi committed:
Update PartsList.md

thewaluigi committed:
Update PartsList.md

thewaluigi committed:
Update PartsList.md

thewaluigi committed:
Update PartsList.md

thewaluigi committed:
Update PartsList.md
